src/implementations/twig/components/gallery/gallery.html.twig

```diff
@@ -49,6 +50,7 @@
 {% set _grid = grid|default(false) %}
 {% set _column = column|default(3) %}
 {% set _ratio = ratio|default('3-2') %}
+{% set _picture_zoom = picture_zoom|default(false) %}
```

Comment:
but why do we add a parameter in all these components? Shouldn't it be just an additional key in the picture component, and then you just pass it along its object?

Reply:
well, ok, just a question: like this in the gallery you will have either all the items animated or none of them, is this the expected behavior? For how i see this we are adding a property to video and picture, it should simply be possible to activate or deactivate this for the single media in the data provided, there is no need to involve the parent templates, but fine.

Reply:
I think that it is expected to act on all images at once. We will confirm in today's meeting